src/pieces/commands/onboarding.py

```diff
@@ -139,15 +135,17 @@ def onboarding_command(**kwargs):
             )
         ]
     }
-    console.print("Welcome to the Pieces CLI")
+    console.print(Markdown("# Welcome to the Pieces CLI\n"
+                           "Remember Anything and Interact with Everything\n"))
```

Comment:
Possible to include this line within the outlined box centered underneath "Welcome to the Pieces CLI" ?

"Remember Anything and Interact with Everything"